.github/workflows/test.yml

```diff
@@ -41,6 +41,10 @@ jobs:
         if: always()
         run: |
           docker logs test-dagger-engine
+      - name: Print Kernel Logs
+        if: always()
```

Comment:
`if: failure()` maybe? but who cares

Reply:
Haha, I actually discovered even on a success run that dmesg showed the fun old `overlayfs: lowerdir is in-use as upperdir/workdir of another mount, accessing files from both mounts will result in undefined behavior.` warning. Which isn't actionable, but mildly concerning, so I suppose there is some limited value in keeping them even in success cases.